### DISCUSSION TITLE
Question: What's 'jujutsu' from?

### DISCUSSION THREAD

**Discussion by dala00** [jj-vcs/jj#3021] [Q&A] [answered]
What's 'jujutsu' from?

- 柔術 (jujutsu)
- 呪術 (jujutsu)
- 呪術廻戦 (jujutsu kaisen)

**Comment by martinvonz** [comment, marked as answer]
https://en.wikipedia.org/wiki/Jujutsu (I would have spelled it "jiujitsu" but then googled it to check the spelling). Renaming the project from 柔術 to 呪術 is a no-op, because we never actually say anywhere exactly what it's meant to refer to :)

**Reply by dala00** [reply]
Thanks!

**Comment by KimotoYanke** [comment]
FYI: It may be better to write "J _ū_ jutsu" or kanji "柔術" in the README.md. Japanese speakers distinguish between ū and u (i.e. pronounced long or short), just as English speakers distinguish between L and R, so developers in Japan are confused about how to pronounce it.

**Reply by thoughtpolice** [reply]
Interesting. I was actually wondering when I read this discussion if we instead might want to adopt "呪術" as the official interpretation; my understanding is that it roughly means "Sorcery" with a kind of neutral sentiment, and many people have said that `jj` is like magic. So maybe it's a bit more fitting.

I think that either way, it would be good to clarify the Japanese pronunciation. I can try sending a PR for it and we can hash it out.

**Reply by thoughtpolice** [reply]
I went ahead and filed https://github.com/martinvonz/jj/pull/3042 to follow up on this. Thanks for the info!
